HelpSource/Classes/UnitTest.schelp

```diff
@@ -29,14 +29,14 @@ UnitTest.gui
 METHOD:: runTest
 Run a single test for methodName.
 
-ARGUMENT:: methodName
-Should have the format code::"TestPolyPlayerPool:test_prepareChildrenToBundle"::
-
 code::
 UnitTest.reset;
 UnitTest.runTest("TestUnitTest:test_assert");
 ::
 
+ARGUMENT:: methodName
+Should have the format code::"TestPolyPlayerPool:test_prepareChildrenToBundle"::
```

Comment:
I think it would be helpful to state that this argument should be a String (i.e. not a Symbol)